dev-libs/libratbag/libratbag-0.13.ebuild

```diff
@@ -0,0 +1,63 @@
+# Copyright 1999-2020 Gentoo Authors
+# Distributed under the terms of the GNU General Public License v2
+
+EAPI=7
+
+PYTHON_COMPAT=( python3_{6,7} )
+
+inherit meson python-r1 udev
```

Comment:
`python-r1` is most likely wrong, and looks to me like you want to use `python-single-r1` instead.

Reply:
I cannot get this working with python-single-r1.  It looks like dev-python/python-evdev[${PYTHON_USEDEP}] is required. dev-python/python-evdev does not take a singles target, and I cannot use PYTHON_USEDEP with python-single-r1.

Reply:
I think I got it working.